frontend/tests/features/wheresMyGene/wheresMyGene.test.ts

```diff
@@ -408,22 +408,20 @@ describe("Where's My Gene", () => {
       );
     });
 
-    // Note: This test could fail if we find more marker genes for embryo hematopoietic cells.
+    // Note: This test could fail if we find more marker genes for the targeted cell.
     // If this happens, just mark the test as skipped and ping @joyceyan
     test(`Should verify copy for cell types with no marker genes`, async ({
       page,
     }) => {
       await goToPage(`${TEST_URL}${ROUTES.WHERE_IS_MY_GENE}`, page);
 
       // Expand embryo tissue
-      await expandTissue(page, "embryo");
+      await expandTissue(page, "saliva");
```

Comment:
@joyceyan the updated tissue and cell for the test are here!

Reply:
weird - this change seems fine though!

Reply:
Awesome! Yeah I checked that saliva leukocyte doesn't have marker genes in staging and prod too 👍